src/GraphQl/Action/EntrypointAction.php

```diff
@@ -67,7 +67,7 @@ public function __invoke(Request $request): Response
             $executionResult = new ExecutionResult(null, [$e]);
         }
 
-        return new JsonResponse($executionResult->toArray($this->debug), $executionResult->errors ? Response::HTTP_BAD_REQUEST : Response::HTTP_OK);
```

Comment:
I would just keep this change. (It looks like Apollo behave in a similar way).

Reply:
If Apollo behave in a similar way, IMO we should keep it.

Reply:
I think their choices are misleading for the GraphQL ecosystem.